src/SignupRouter.js

```diff
@@ -10,7 +10,7 @@ import Signup from './components/signup/SignupState'
 import SigninInfo from './components/signin/SigninInfo'
 import Blurred from './components/common/view/Blurred'
 import Welcome from './components/auth/login/WelcomeScreen'
-import InviteWelcome from './components/inviteWelcome/InviteWelcome'
+import InviteWelcome from './components/auth/Welcome'
```

Comment:
We are keeping the InviteWelcome Exactly as it is.

Reply:
@guodevguru  yes please revert this